.github/workflows/rerun.yml

```
@@ -0,0 +1,22 @@
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'The ID of the workflow run to rerun'
+        required: true
+      runner:
+        description: 'The runner to use for the rerun'
+        required: true
+
+jobs:
+  rerun:
+    runs-on: ${{ inputs.runner }}
+    steps:
+      - name: rerun ${{ inputs.run_id }}
+        env:
+          GH_REPO: ${{ github.repository }}
+          GH_TOKEN: ${{ github.token }}
+          GH_DEBUG: api
+        run: |
+          gh run watch ${{ inputs.run_id }} > /dev/null 2>&1
```

```suggestion
          gh run watch ${{ inputs.run_id }}
```

(should help debug by showing real-time progress plus any errors, instead of suppressing all output)

edit: checked, no sensitive data emitted